docs/guide/contributing.rst

```diff
@@ -0,0 +1,92 @@
+Contributing to hipscat
+===============================================================================
+
+Find (or make) a new GitHub issue
+-------------------------------------------------------------------------------
+
+Add yourself as the assignee on an existing issue so that we know who's working 
+on what. If you're not actively working on an issue, unassign yourself.
+
+If there isn't an issue for the work you want to do, please create one and include
+a description.
+
+You can reach the team with bug reports, feature requests, and general inquiries
+by creating a new GitHub issue.
+
+Create a branch
+-------------------------------------------------------------------------------
+
+It is preferable that you create a new branch with a name like 
+``issue/##/<short-description>``. GitHub makes it pretty easy to associate 
+branches and tickets, but it's nice when it's in the name.
+
+Setting up a development environment
```

Comment:
Tiny thing, consistency of heading tenses (though technically, "Testing" also is also an -ing ending, but "Setting" is the only one that jumped out at me). This could be just a me thing, so feel free to ignore.